<ui version="4.0">
 <class>SearchUi</class>
 <widget class="QDialog" name="SearchUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>152</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="findLA">
     <property name="text">
      <string>&amp;Find:</string>
     </property>
     <property name="buddy">
      <cstring>findCO</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="findCO">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxCount">
      <number>666</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtTop</enum>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="replaceLA">
     <property name="text">
      <string>Replace &amp;with:</string>
     </property>
     <property name="buddy">
      <cstring>replaceCO</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="replaceCO">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxCount">
      <number>666</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAtTop</enum>
     </property>
     <property name="duplicatesEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="caseCB">
     <property name="text">
      <string>Case &amp;sensitive[[search]]</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2" colspan="2">
    <widget class="QCheckBox" name="wordsCB">
     <property name="text">
      <string>Match w&amp;hole words only</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="3">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="findPB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Find &amp;Next</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="replacePB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Replace</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" rowspan="2">
    <widget class="QPushButton" name="replaceallPB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Replace &amp;All</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QCheckBox" name="backwardsCB">
     <property name="text">
      <string>Search &amp;backwards</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" rowspan="2">
    <widget class="QPushButton" name="closePB">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findCO</tabstop>
  <tabstop>replaceCO</tabstop>
  <tabstop>caseCB</tabstop>
  <tabstop>wordsCB</tabstop>
  <tabstop>backwardsCB</tabstop>
  <tabstop>findPB</tabstop>
  <tabstop>replacePB</tabstop>
  <tabstop>replaceallPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
